WebDevStudios/gatsby-starter-wordpress-graphql
# React Friendly Styles

Closes #10 

## Description
Based on what @martinDolan started, I created basic styles based on https://theme-ui.com/

## Screenshot
![screenshot](https://dl.dropbox.com/s/r08khikuzmi72lg/screenshot%202019-09-11%2014.51.13.gif?dl=0)

## Steps for testing

Visit: https://deploy-preview-31--gatsby-starter-wordpress-graphql.netlify.com/

and check out this branch [feature/#10-style-library](https://github.com/WebDevStudios/gatsby-starter-wordpress-graphql/tree/feature/%2310-style-library)
